apps/user-portal/src/locales/ta/common.ts

```diff
@@ -35,12 +35,15 @@ export const common: Common = {
     claim: "கோர்",
     completed: "பூரணப்படுத்தப்பட்டவை",
     configure: "கட்டமை",
+    contains: "கொண்டுள்ளது", // TODO: translate
     continue: "தொடர்",
     createdOn: "உருவாக்கப்பட்ட தினம்",
     delete: "அழி",
     description: "விபரம்",
     deviceModel: "கருவி மாதிரி",
     edit: "திருத்து",
+    endsWith: "Ends with", // TODO: translate
+    equals: "Equals", // TODO: translate
```

Comment:
Shall we get these translated?

Reply:
@thivi Can you please translate these in your next PR.

Reply:
Inform @thivi to translate them on his next PR.